searchlib/src/tests/attribute/enumstore/enumstore_test.cpp

```diff
@@ -22,41 +22,61 @@ struct OrderedDoubleEnumStore {
     static constexpr Type type = Type::BTREE;
 };
 
-struct UnorderedDoubleEnumStore {
+struct HybridDoubleEnumStore {
     using EnumStoreType = DoubleEnumStore;
     static constexpr Type type = Type::BTREE_AND_HASH;
 };
 
+struct UnorderedDoubleEnumStore {
```

Comment:
Consider renaming _Unordered_ -> _Hash_ and _Ordered_ -> _BTree_. Applies for other structs in this file as well.